cmd/describe/cluster/cmd.go

```diff
@@ -144,7 +158,7 @@ func run(_ *cobra.Command, argv []string) {
 		"Console URL:   %s\n"+
 		"Nodes:         Master: %d, Infra: %d, Compute: %d\n"+
 		"Region:        %s\n"+
-		"State:         %s\n"+
+		"State:         %s (%s)\n"+
```

Comment:
If `phase` is an empty string (like in the case of `state == installing, dnsready == true`), this will show something like `State: Installing ()`

Reply:
updated